src/sec_edgar_bulker/core/exceptions.py

```
@@ -0,0 +1,47 @@
+    """Raised when there is an error parsing a file."""
+    pass
+
+class DownloadModeError(Exception):
+    """Raised when there is an error with the download mode."""
+    def __init__(self, message, details=None):
+        super().__init__(message)
+        self.details = details
+
+class InvalidFormTypeError(Exception):
+    """Raised when an invalid form type is encountered."""
+    pass
```

Ruff (0.8.2)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)